### DISCUSSION TITLE
On Node hover, tooltip show

### DISCUSSION THREAD

**Discussion by sunanda35** [xyflow/xyflow#1941] [New Features]
when the mouse hovers over the node, its specific data should show as a tooltip. I needed this feature.

**Comment by moklick** [comment]
Hey @sunanda35 

there is no Tooltip component built-in but you can use `onNodeMouseEnter` and `onNodeMouseLeave` for this.

**Reply by sunanda35** [reply]
i'm requesting this feature to come up with tooltip feature

**Reply by sunanda35** [reply]
@moklick can I do that? I want to develop this feature.

**Reply by Krithika3** [reply]
@moklick @sunanda35 is there a plan to develop this feature?
